zeppelin-web/src/app/visualization/builtins/visualization-table-setting.html

```diff
@@ -17,6 +17,12 @@
     <span style="vertical-align: middle; display: inline-block; margin-top: 3px;">Table Options</span>
     <span style="float: right;">
        <div class="btn-group" role="group" aria-label="...">
+         <div type="button" ng-click="applyTableOption()"
+                       uib-tooltip="Apply new setting" tooltip-placement="left"
+                       class="btn btn-default" style="font-size: 11px; padding: 2px 5px 2px 5px;">
+           <i class="fa fa-check-square" aria-hidden="true"></i>
```

Comment:
this looks useful but a checked box button might be a bit confusing as "check all box" perhaps?

Reply:
Maybe "save" button would be better, WDYT?
![new-button](https://user-images.githubusercontent.com/6136993/47253832-a6ca6b80-d461-11e8-8e92-bba2c629563e.png)

Reply:
ping @felixcheung

Reply:
ping @felixcheung

Reply:
@felixcheung Look again, please :)

Reply:
hmm, I guess either is ok :)

so I'm wondering with this behavior change, would it be easy to forget to "apply"? would it be possible to pop a tooltip when anything is changed to direct user to click "apply all"?

Reply:
@felixcheung table settings have just three options, so I suppose it's not really necessary, moreover this behavior is not supported in more complexity settings in `advanced-transformation-settings`.

Reply:
could you clarify? I'm not suggest to change any behavior - just pop a tooltip when one of the three option is changed, since, if I understand, any change applies immediately. now, with your change, it would seem to the user that "it doesn't do anything"
?

Reply:
Now changes apply only after page refreshing, my PR saves this logic, but also provides button with apply changes function. As for me, this tooltip is not necessary, because there are only three options in list.

Reply:
your screencast shows multiple changes are applied only after save is clicked, and it refreshes the visual. am I understanding you right?
https://user-images.githubusercontent.com/6136993/51320121-ecea9780-1a6f-11e9-850f-fcaf061ff5f3.gif

Reply:
I've just tried to show new functionality, you also could just refresh the page after cliking on checkboxes and changes will be applied.

Reply:
I just tried to show the new functionality, you can also just refresh the page after clicking on the checkboxes, and the changes will be applied.